sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/datetimeExpressions.scala

```diff
@@ -1008,17 +1008,17 @@ case class UnixTimestamp(
     copy(timeExp = newLeft, format = newRight)
 }
 
-case class GetTimestampNTZ(
+/**
+ * Gets timestamps from strings using given pattern.
```

Comment:
Not only from strings but from dates, for instance. And the given pattern is optional since it is not used when the input is a date.

Reply:
Updated